### ISSUE TITLE
Technology behind Atom editor

### ISSUE THREAD

**Issue by jbranchaud** [jbranchaud/splitting-atoms#3]
We need to put together a document detailing all the technology behind Atom.

[The Nucleus of Atom](http://blog.atom.io/2014/02/26/the-nucleus-of-atom.html) sheds light on some of this.

Other technologies to note:
- [node-webkit](https://github.com/rogerwang/node-webkit)
- [space-pen](https://github.com/atom/space-pen)
- [flexbox intro](http://css-tricks.com/snippets/css/a-guide-to-flexbox/) and [using flexbox](https://developer.mozilla.org/en-US/docs/Web/Guide/CSS/Flexible_boxes)

_What are we missing?_

**Comment by karan** [comment]
Also, maybe list a few other projects that are made using some combination of technologies that Atom uses. It will give a good idea of what all is possible with what Atom is made of.

I can name a few Markdown editors built on top of node webkit - Haroopad and Mou.

**Comment by jbranchaud** [comment]
@karan Good point, here is a [massive list of stuff built with node-webkit](https://github.com/rogerwang/node-webkit/wiki/List-of-apps-and-companies-using-node-webkit).

**Comment by karan** [comment]
That is massive indeed! :facepunch: 

But maybe limit to only well known (or well executed) stuff?
